www/src/components/shared/pullquote.js

```diff
@@ -37,6 +37,14 @@ const PullquoteRoot = styled(`blockquote`)`
   }
 `
 
+const Citation = styled(`cite`)`
+  display: block;
+  text-align: right;
```

Comment:
A nit. The alphabetical order of props, so `text-align` goes at the end. 

I also do not care in what order the properties like `font-weight` and `font-style` goes. Only the first character counts for me.

Reply:
Lol, done. Thanks for enforcing.